PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -191,6 +201,10 @@ def miniAOD_customizeCommon(process):
     #
     ## PU JetID
     process.load("RecoJets.JetProducers.PileupJetID_cfi")
+    task.add(process.pileupJetId)
+    task.add(process.pileupJetIdCalculator)
+    task.add(process.pileupJetIdEvaluator)
```

Comment:
How was it derived? I looked inside the cfi file at what the load attaches to the process knowing that those would be the things setup to run unscheduled. Then I made sure the same modules were attached to a task so the list of modules configured to run unscheduled remained the same. (And I ran tests that checked for mistakes)

In this case this is a cfi which originally had no Sequence in it. Do we want Tasks defined in cfi files? I am still thinking about this and we can discuss the rest your question more.